networkx/algorithms/planarity.py

```diff
@@ -24,8 +25,29 @@ def check_planarity(G, counterexample=False):
         If the graph is planar `certificate` is a PlanarEmbedding
         otherwise it is a Kuratowski subgraph.
 
+    Examples
+    --------
+
+    This function can be considered as the primary interface for the planar embedding tools since
+    -if the graph is planar- it returns a PlanarEmbedding which can be then used to call other methods of this class.
+    Consider the following example:
+
+    >>> G = nx.Graph([(0,1),(0,2)])
+    >>> isPlanar, P = nx.check_planarity(G)
+    >>> print(isPlanar)
+    True
+
+    Once the PlanarEmbedding object is created using `check_planarity()` as above, the other features of the class can be used safely.
+    Let's see this on action by extending our initial example:
+
+    >>> G = nx.Graph([(0,1),(0,2)])
+    >>> isPlanar, P = nx.check_planarity(G)
+    >>> P.get_data()
+    {0: [1, 2], 1: [0], 2: [0]}
+
```

Comment:
Here it might be good to also include an example when the graph is non-planar, e.g. with a complete_graph. I actually find the API of this function w/ the `counterexample` kwarg a bit confusing (it returns `None` if `is_planar` is False and `counterexample=False`?).

No need to tackle that in this PR though - feel free to leave it for a followup PR if you'd like to instead focus on what's already here!

Reply:
Thank you @rossbar , I made the changes you suggested. I will handle this one in a follow up PR and let you know.